README.md

```diff
@@ -129,8 +129,10 @@ With plugin.xml containing;
         - Diagnostics and code actions
             ```xml
             <extensions defaultExtensionNs="com.intellij">
-                <inspectionToolProvider implementation="org.wso2.lsp4intellij.contributors.inspection.LSPInspectionProvider"
```

Comment:
Looks like i've missed this part. 'll check it ASAP

Reply:
Sure :+1: